satellite/satellitedb/migrate.go

```diff
@@ -1057,6 +1057,35 @@ func (db *DB) PostgresMigration() *migrate.Migration {
 						WHERE credits_earned_in_cents=0;`,
 				},
 			},
+			{
+				// Creating owner_id column for project.
+				// Removing projects without project members
+				// And populating this column with first project member id
+				Description: "Create partial index for user_credits table",
+				Version:     49,
+				Action: migrate.SQL{
+					`ALTER TABLE projects
+						ADD COLUMN owner_id BYTEA;
+
+                    DELETE FROM projects WHERE id NOT IN (
+                        SELECT project_id
+                            FROM project_members
+                                LEFT OUTER JOIN projects
+                                    ON projects.id = project_members.project_id);
+
+					UPDATE projects as proj
+                        SET owner_id = 
+                            (SELECT member_id
```

Comment:
If I'm not mistaken this subquery could be simpler

```sql
SELECT member_id
FROM project_members
WHERE project_id = proj.id
ORDER BY created_at ASC
LIMIT 1;
```

I've tried both doing a small simulation in http://sqlfiddle.com

Selecting Postgres 9.3
Using this schema

```sql
CREATE TABLE projects (
  id INT NOT NULL,
  UNIQUE(id)
);

INSERT INTO projects (id) VALUES (0);
INSERT INTO projects (id) VALUES (1);

CREATE TABLE members (
  member_id INT NOT NULL,
  project_id INT NOT NULL REFERENCES projects(id), 
  created_at timestamp with time zone NOT NULL,
  PRIMARY KEY (member_id, project_id)
);

INSERT INTO members (member_id, project_id, created_at) VALUES (1, 0, '2000-09-28 03:00:00');
INSERT INTO members (member_id, project_id, created_at) VALUES (1, 1, '2002-09-28 03:00:00');
INSERT INTO members (member_id, project_id, created_at) VALUES (2, 0, '2001-09-28 03:00:00');
INSERT INTO members (member_id, project_id, created_at) VALUES (2, 1, '2003-09-28 03:00:00');
```

And the following queries:

```sql
-- current subquery

SELECT member_id
FROM members
WHERE project_id = 1
GROUP BY member_id
ORDER BY MAX(created_at) ASC
LIMIT 1;


-- new subquery

SELECT member_id
FROM members
WHERE project_id = 1
ORDER BY created_at ASC
LIMIT 1;

```